elementary/clients/gcs/client.py

```diff
@@ -43,22 +44,38 @@ def send_report(
             logger.info("Uploaded report to GCS.")
             if self.config.update_bucket_website:
                 bucket_report_folder_path = bucket_path.dirname(bucket_report_path)
+                bucket_name = (
+                    bucket_path.join_path([bucket_report_folder_path, "index.html"])
+                    if bucket_report_folder_path
+                    else "index.html"
+                )
                 bucket.copy_blob(
                     blob=blob,
                     destination_bucket=bucket,
-                    new_name=bucket_path.join_path(
-                        [bucket_report_folder_path, "index.html"]
-                    )
-                    if bucket_report_folder_path
-                    else "index.html",
+                    new_name=bucket_name,
+                )
+                bucket_website_url = self.get_bucket_website_url(
+                    destination_bucket=self.config.gcs_bucket_name,
+                    bucket_name=bucket_name,
                 )
                 logger.info("Updated GCS bucket's website.")
         except google.cloud.exceptions.GoogleCloudError as ex:
             logger.exception("Failed to upload report to GCS.")
             if self.tracking:
                 self.tracking.record_cli_internal_exception(ex)
-            return False
-        return True
+            return False, bucket_website_url
+        return True, bucket_website_url
+
+    def get_bucket_website_url(
+        self, bucket_name: str, destination_bucket: Optional[str] = None
+    ) -> Optional[str]:
```

Comment:
You are right about the presigned URL, but this is how we guide our users for now because anyone with the signed link can access the presigned URL, so we decided not to do it for now.
We can talk about this, but this is our behaviour since day 1 and I don't think this is part of the task scope 🙂